inference-engine/src/inference_engine/ie_system_conf.cpp

```diff
@@ -90,16 +90,25 @@ bool checkOpenMpEnvVars(bool includeOMPNumThreads) {
 #if defined(__APPLE__)
 // for Linux and Windows the getNumberOfCPUCores (that accounts only for physical cores) implementation is OS-specific
 // (see cpp files in corresponding folders), for __APPLE__ it is default :
-int getNumberOfCPUCores() { return parallel_get_max_threads();}
+int getNumberOfCPUCores(bool) { return parallel_get_max_threads();}
 #if !((IE_THREAD == IE_THREAD_TBB) || (IE_THREAD == IE_THREAD_TBB_AUTO))
-std::vector<int> getAvailableNUMANodes() { return {0}; }
+std::vector<int> getAvailableNUMANodes() { return {-1}; }
```

Comment:
aligned to the default fallback values that the TBB produces (e.g. when hwloc fails)